x-pack/plugins/secops/public/components/link_to/redirect_to_hosts.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+export const RedirectToHostsPage = () => {
+  return <Redirect to={`/hosts`} />;
+};
+
+export const getHostsUrl = () => {
+  return `#/link-to/hosts`;
+};
```

Comment:
Single quotes since there are no variables and you can do an implicit return?

```ts
export const getHostsUrl = () => '#/link-to-hosts';
```

Reply:
It will change soon enough when I add the KQL